backend/app/Http/Controllers/SampleController.php

```diff
@@ -503,14 +514,35 @@ public function filterDataImport($filter,$value){
             {
                 return "GARUT";
             }
+        }elseif($filter=='tanggal')
+        {
+            if($value=='23/4//67')
+            {
+                return null;
+
+            }elseif($value=='33/05/1951')
+            {
+                return null;
+
+            }elseif($value='11/058/1966')
```

Comment:
Kang @nurisakbar yang ini typo-kah?

```suggestion
            }elseif($value='11/05/1966')
```

Reply:
iya kang, typo dari operator nya waktu input data ke excel